swan-lake/development-tutorials/configurability/provide-values-to-configurable-variables.md

```diff
@@ -10,13 +10,41 @@ intro: You can provide values for configurable variables through multiple method
 
-1. **Command-line arguments:** Configure the values through command-line arguments executed when running the Ballerina program. 
+1. **Environment variables:** Configure the values through separate environment variables for each 
+   configurable variable.
 
-2. **Configuration files:** Define the values using the [TOML syntax](#toml-syntax) and configure them through configuration files. 
+2. **Command-line arguments:** Configure the values through command-line arguments executed when running the Ballerina 
+program. 
 
-3. **Environment variables:** Define the values using the [TOML syntax](#toml-syntax) and configure them through environment variables.
+3. **Configuration using TOML content:**
+   - Define the values using the [TOML syntax](#toml-syntax) and configure them through 
+     configuration files.
+   - Define the values using the [TOML syntax](#toml-syntax) and configure them through
+     environment variables.
 
-    >**Note:** Providing configuration values through separate environment variables for each configurable variable is currently not supported.
+## Provide via environment variables
+
+Use the syntax below to provide values for the variables through command-line arguments.
+
+```
+BAL_CONFIG_VAR_key=value
+```
+
+>**Note** The configurable value provided through an environment variable is expected to be in the `toString()` representation of the intended value. Currently, the environment-variable-based configuration supports only the configurable variables of types `int`, `byte`, `float`, `boolean`, `string`, `decimal`, `enum`, and `xml`.
+
+The following examples explain how to provide environment variables to configure variables of specific Ballerina types.
+
+| Ballerina type | Ballerina example                                                                                                                                                                                             | Setting environment variable for Windows                                          | Setting environment variable for Linux/macOS                                            |
+|----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
+| boolean        | <code>configurable boolean isAdmin = ?; </code>                                                                                                                                                               | `$ set BAL_CONFIG_VAR_ISADMIN=true` <br> or <br> `$ set BAL_CONFIG_VAR_ISADMIN=1` | `$ export BAL_CONFIG_VAR_ISADMIN=true` <br> or <br> `$ export BAL_CONFIG_VAR_ISADMIN=1` |
```

Comment:
Is there a reason to use <code> tags here? You should be able to use back ticks.

Reply:
Some parts contain multiline code that includes Balerina record definitions. In earlier versions, it didn't render properly when we used backticks.